src/Raku/Grammar.nqp

```diff
@@ -5391,7 +5390,7 @@ Rakudo significantly on *every* run."
     }
 
     token doc-block:sym<lines> {
-        [^^ \h* [ <![=]> \N+ | '=' \W \N* ] \n?]+
```

Comment:
@lizmat This line worth paying attention anyway because \N+ is clearly an error since it makes single-char lines illegal in a way.